change/@ni-nimble-components-60beeec6-9cec-44db-a7d5-8588b0862986.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Your PR description uses the "Fixes" keyword to link to the related issue, which will [result in that issue being closed](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) when the PR completes. Since the framework integration is still to come, could you change the description not to use a keyword?

Reply:
Sure, changed.